charts/kubeclarity/values.yaml

```diff
@@ -41,6 +41,36 @@ kubeclarity:
 
   service:
     type: ClusterIP
+    annotations: {}
+    # Port on which Kubeclarity server is listening
+    port: 8080
+    # Target port to which the service should be mapped to
+    targetPort: 8080
```

Comment:
The target port is defined by the kubeclarity deployment (i.e. its the port that the kubeclarity pod is exposing) and this isn't something that is currently configurable so setting this to anything other than 8080 will break the service.

If we want to be able to change this like we do for the runtime scan port, then I think we need a configuration outside of "service" for listenPort and then we need to template it into the deployment.yaml.

IMO Its not something that I think needs to be configurable, the fact that kubeclarity is being exposed on 8080 behind the service port (which is user facing and can be/should configurable) is pretty much an entirely internal detail.